other/rootfs/jbin/post.sh

```diff
@@ -0,0 +1,17 @@
+binpack=/jbin/binpack
+
+# uicache loader app
+$binpack/bin/mkdir -p /var/.palera1n
+$binpack/bin/rm -rf /var/.palera1n/loader.app
+$binpack/bin/cp -R /jbin/loader.app /var/.palera1n
+$binpack/usr/bin/uicache -p /var/.palera1n/loader.app
+
+# link dyld if using semi tethered (iOS 16)
+if [ -d "/System/Cryptexes/OS/System/Library/Caches/com.apple.dyld" ]; then
+    if [ ! -d "/System/Library/Caches/com.apple.dyld" ]; then
+        $binpack/bin/ln -s /System/Cryptexes/OS/System/Library/Caches/com.apple.dyld /System/Library/Caches/
+    fi
+fi
+
+echo "[post.sh] done"
+exit
```

Comment:
missing end of file new line
```suggestion
exit

```